ios/src/platform_ios.mm

```diff
@@ -153,20 +153,18 @@ bool isContinuousRendering() {
 
     for (id fallback in fallbacks) {
 
-        UIFont* font = [UIFont fontWithName:fallback size:1.0];
+        handles.emplace_back([fallback]() {
 
-        size_t dataSize = 0;
-        auto cdata = loadUIFont(font, &dataSize);
+            std::vector<char> data;
 
-        if (!cdata) { continue; }
-
-        FontSourceHandle fontSourceHandle = [cdata, dataSize](size_t* _size) -> unsigned char* {
-            *_size = dataSize;
-
-            return cdata;
-        };
-
-        handles.push_back(fontSourceHandle);
+            UIFont* font = [UIFont fontWithName:fallback size:1.0];
+            size_t dataSize = 0;
+            auto cdata = loadUIFont(font, &dataSize);
```

Comment:
Just for the sake of documenting this PR, after some digging with @hjanetzek here are some numbers will full optimization we've found with the different available options:
```
1199.566895ms for assign with reserve
1202.317993ms for assign without reserve
1327.153198ms for std::copy with reserve
1450.960205ms for std::copy without reserve
1263.508057ms for insert with reserve
889.072998ms for insert without reserve
395.985931ms without the copy
```

So, further work here to improve interfacing with alfons would save us half the time.